vocabularies/examples/com.sap.vocabularies.UI.v1.UserInteraction-sample.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/UI.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.UI.v1" Alias="UI" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Common.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Common.v1" Alias="Common" />
+  </edmx:Reference>
+  <edmx:Reference Uri="/sap/opu/odata4/product.xml">
+    <edmx:Include Namespace="com.sap.gateway.entities.product" Alias="Product" />
+  </edmx:Reference>  
+  <edmx:DataServices>
+    <Schema Namespace="userinteraction.sample" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+      <!-- Certain user interactions cannot be handled by one reqeust, because they follow the pattern "user triggers, server offers, user chooses".
+           For example, the action "exchange product in a sales order item" is triggered by the user on a sales order item
+           (POST ~/SalesOrderItems(...)/ExchangeProduct), the server returns the item with an inlined collection of alternative products,
+           from which the user must then choose, and the request must be repeated with the chosen key
+           (POST ~/SalesOrderItems(...)/ExchangeProduct?NewProduct='...').
```

Comment:
So how is it expected show in the UI. User performs an action the result is shown in a Dialog with selectable list is it? What if the use case support selecting more than one item

Reply:
Yes, but this still requires alignment with the UI frameworks. (That's why this PR is still a draft.)

Reply:
Ok